src/main/java/springsamurais/toyswapbackend/model/User.java

```diff
@@ -1,3 +1,82 @@
 package springsamurais.toyswapbackend.model;
 
-public class User {}
+import jakarta.persistence.*;
+import org.antlr.v4.runtime.misc.NotNull;
+
+
+import java.util.List;
+
+@Entity
+//@Table(name = "user")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    private String name;
+
+    @Column(unique = true)
+    @NotNull
+    private String nickname;
+
+    private String location;
+
+
+    @Column
```

Comment:
Correct me if mistaken @bm298 . This is a @OnetoMany relation.

Reply:
Yes, one user has many posts - so should be @OneToMany.